snowtrick/assets/app.js

```diff
@@ -6,7 +6,116 @@
  */
 
 // any CSS you import will output into a single css file (app.css in this case)
-import './styles/app.css';
+import './styles/app.scss';
 
 // start the Stimulus application
-import './bootstrap';
+
+document.addEventListener('DOMContentLoaded', () => {
+// Functions to open and close a modal
+    function openModal($el) {
+        $el.classList.add('is-active');
+    }
+
+    function closeModal($el) {
+        $el.classList.remove('is-active');
```

Comment:
Codacy has a fix for the issue: [Replace `····$el.classList.remove('is-active'` with `$el.classList.remove("is-active"`](https://app.codacy.com/gh/trousse/OC_Projet6/pullRequest?prid=10057695)
```suggestion
    $el.classList.remove("is-active");
```